pallets/liquid-staking/src/mock.rs

```diff
@@ -129,12 +133,37 @@ impl Config for XcmConfig {
     type AssetClaims = PolkadotXcm;
 }
 
+pub struct SystemParachainAsSuperuser<Origin>(PhantomData<Origin>);
```

Comment:
duplicate several times, maybe we can refact a bit clean

Reply:
also we may not really need such superuser control from other parachain, we can do it from relaychain if nessessary

Reply:
This is mock config which we need for cumulus_pallet_xcmp_queue